src/frontend/src/Modal.js

```diff
@@ -41,6 +41,13 @@ export default function Modal() {
   };
 
   const resetAuthModal = () => {
+    // Reset focus and screen reader content
+    const allFocuses = document.querySelectorAll("a, button, input");
+    allFocuses.forEach((element) => {
+      element.removeAttribute("aria-hidden");
+      element.removeAttribute("tabindex");
+    });
+
```

Comment:
Wouldn't this remove all the tabindexes? So they can't be tabbed anymore after the user closes modal?

Reply:
Pushed a new commit to fix this. Thanks for catching it!